packages/KInlineEdit/KInlineEdit.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    id="editable-wrapper"
+    class="k-inline-edit">
+    <input
+      v-if="isEditing"
+      ref="input"
+      :style="styles"
+      v-model="inputText"
+      class="k-input"
+      @blur="handleSave"
+      @keyup.enter="handleSave">
+    <div
+      id="element-content-wrapper"
+      @click="handleClick">
+      <slot v-if="!isEditing" />
+    </div>
+  </div>
+</template>
+
+<script>
+// Styles we want copied from the element
+const STYLES = {
+  fontSize: 'font-size',
+  fontWeight: 'font-weight',
+  fontFamily: 'font-family',
+  color: 'color',
+  margin: 'margin',
+  padding: 'padding'
+}
+
+export default {
+  name: 'KInlineEdit',
+  data () {
+    return {
+      isEditing: false,
+      inputText: '',
+      styles: {}
+    }
+  },
+  computed: {
+    listeners () {
```

Comment:
this isn't being used?

Reply:
nope, old code, not needed with the `@blur` in the template